source/manual/alerts/data-sync.html.md

```diff
@@ -5,7 +5,7 @@ section: Icinga alerts
 layout: manual_layout
 parent: "/manual.html"
 old_path_in_opsmanual: "../opsmanual/2nd-line/alerts/data-sync.md"
-last_reviewed_on: 2017-02-17
+last_reviewed_on: 2017-07-25
```

Comment:
This should probably be `2017-08-25` (or `2017-08-29` by now)

Reply:
Ooops! Will fix. Thanks